### PR TITLE
feat(button): add dropdown-indicator option (VIV-1741)

### PR DESCRIPTION
**Pull request by RichardHelm**

### CODE REVIEW

File: libs/components/src/lib/button/button.scss [hunk 1]
```diff
@@ -82,20 +83,6 @@
...
-		padding-inline: 0;
-		place-content: center;
-
-		@supports (aspect-ratio: 1) {
-			aspect-ratio: 1;
-		}
-
-		@supports not (aspect-ratio: 1) {
-			inline-size: var(#{variables.$block-size});
-		}
```
Comment (rachelbt) [inline libs/components/src/lib/button/button.scss:96]: why did you removed this?
This is better then adding padding

File: libs/components/src/lib/button/button.scss [hunk 2]
```diff
@@ -228,15 +212,32 @@ slot[name='icon'] {
...
+	gap: var(#{variables.$icon-gap});
+
+	.control.stacked & {
+		flex-direction: column;
+	}
+}
+
+.chevron {
+	flex-shrink: 0;
+}
```
Comment (rachelbt) [inline libs/components/src/lib/button/button.scss:239]:
```suggestion
```
this is not needed - its in the chevron file already

**Review by rachelbt — COMMENTED** [review]
nice 👍

File: libs/components/src/lib/button/button.scss (hunk 1 above)
Comment (RichardHelm) [reply libs/components/src/lib/button/button.scss:96]: When icon-only button has chevron, it is no longer square

**Review by rachelbt — APPROVED** [review]
⚡

**Review by rachelbt — APPROVED** [review]

**Review by rachelbt — APPROVED** [review]
I think it looks better now with 16px for the chevron in normal

**Pull request merged**